package.json

```diff
@@ -9,40 +9,52 @@
     "lint": "next lint"
   },
   "dependencies": {
```

Comment:
can you also help remove the unused depenencies? i get these when running `npx depcheck`

```
Unused dependencies
* @aws-amplify/form-generator
* @aws-amplify/model-generator
* core-js
* glob
* install
* lucide
* npm
* react-google-recaptcha
* rimraf
Unused devDependencies
* @aws-amplify/backend
* @aws-amplify/backend-cli
* @types/node
* @types/react-dom
* @types/react-google-recaptcha
* postcss
* prettier-plugin-tailwindcss
* typescript
Missing dependencies
* eslint-config-next: ./.eslintrc.json
```

Reply:
![image](https://github.com/user-attachments/assets/d041a2c0-296f-42e1-87d5-45d886fbebeb)